README.md

```diff
@@ -35,6 +38,7 @@ brokernode# buffalo test
 ```
 
```

Comment:
Do we need some sort of database.dev.yml file as well?

Reply:
This worked for me. The yml file in the `dev-dbs` branch looks to be the same as what's on master.

Reply:
They are some differences.  Configuration for test is the same, but development and production are different.

Reply:
still a little weird how the tests pass on travis, but not locally. Ill try copy and pasting the database.yml and see if that fixes it for me